pkg/aspect/config/accessLogs.proto

```diff
@@ -21,54 +21,63 @@ option go_package="config";
 // Configures an individual access-logs aspect.
 //
 // Example usage:
-// kind: access-logs
-// params:
-//    log_name: "logs/access_log"
-//    attributes: "source.ip","host.URL","
-//    log_format: "COMMON"
+//     kind: access-logs
+//     params:
+//        log_name: "logs/access_log"
+//        log:
+//            log_format: CUSTOM
```

Comment:
since we don't support CUSTOM at the moment, we should probably use a different example here.